examples/linux64/hello-make/linux64.ld

```diff
@@ -0,0 +1,9 @@
+MEMORY code LOCATION=0x40000 SIZE=0x10000 {
+    SECTION(reset)
```

Comment:
This `reset` section can be removed. Note that you may want to add the ENTRY to main as well.

Reply:
> This reset section can be removed. 

Ack. Was originally copied verbatim from the "hello" sample, but makes sense to remove any unneeded stuff to not confuse readers.

Reply:
Yes, agreed, I'm removing it from the other sample code as well, to reduce clutter. The reset section was needed before the entry point option.

Reply:
> Note that you may want to add the ENTRY to main as well.

I pass it on the command line instead. The whole idea of this example is to show how to do things from command-line interface.

For as long as this "memory map layout" files aren't compatible with GNU ld "linker scripts", I'd personally prefer to avoid using them too much ;-). (I know that "ENTRY(sym)" syntax is exactly compatiable with GNU ld, thanks for that.)

Reply:
Ah, okay, I missed that, thanks!